src/Domain/LeanCode.DomainModels.EF/PropertiesConfigurationBuilderExtensions.cs

```diff
@@ -0,0 +1,44 @@
+using LeanCode.DomainModels.Model;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace LeanCode.DomainModels.EF;
+
+public static class PropertiesConfigurationBuilderExtensions
+{
+    public static PropertiesConfigurationBuilder<SUlid<T>> AreTypedId<T>(this PropertiesConfigurationBuilder<SUlid<T>> builder)
```

Comment:
Why `AreTypedId` rather than `IsTypedId`?

Reply:
This method is used to configure convention, so the meaning here is `configurationBuilder.Properties<SUlid<T>>().AreTypedId()` is configured context wise, not just for a property, thus the plural here